spamostack/keeper.py

```diff
@@ -119,6 +120,10 @@ def get(self, client_name, resource_name, param=None, func=None,
                     else:
                         probe = getattr(el, param)(*args, **kwargs)
 
+                    if (resource_name == "flavors"
```

Comment:
I'm agree with the use of the defined prefix for all resources to be able to remove them whenever we want.